src/components/Button/Button.tsx

```diff
@@ -18,7 +18,8 @@ function Button({ text, selected, onClick, value, size = 'medium' }: ButtonProps
     <button className={classNames({
       'button': true,
       'button--selected': selected,
-      'button--small': size === 'small'
+      'button--small': size === 'small',
+      'button--send': size === 'large'
```

Comment:
so If the size of button is large, and what define the size is the word large, why did you name the variation send?
button--large (as this size can be used for buttons that don't send anything).